src/pages/Info/InfoPages.css

```
@@ -0,0 +1,466 @@
+  color: #ff4d4f;
+  width: 48px;
+  height: 48px;
+  margin-bottom: 20px;
+}
+
+.thanks-name {
+  font-size: 2.5rem;
+  font-weight: 800;
+  color: #55C5F1;
+  margin: 16px 0;
+  font-family: 'Ephesis', cursive, var(--font-family); /* Use a more elegant font if available */
```

Stylelint (17.10.0)

[error] Expected no quotes around "Ephesis" (font-family-name-quotes)

(font-family-name-quotes)